images/php-laravel/supervisord.conf

```diff
@@ -0,0 +1,11 @@
+[supervisord]
+nodaemon=true
+
+[program:laravel]
```

Comment:
Needs to actually run 2 processes, not 1: 
1) Apache Web Server
2) PHP-FPM

Then you need to configure apache web server to reverse proxy to php-fpm.

Reply:
Logs need to all go to stdout, too